src/appleseed-max-impl/appleseedinteractive/interactiverenderercontroller.h

```diff
@@ -30,6 +30,49 @@
 
 // appleseed.renderer headers.
 #include "renderer/api/rendering.h"
+#include "renderer/api/project.h"
+#include "renderer/api/scene.h"
+
+// appleseed.foundation headers.
+#include "foundation/utility/autoreleaseptr.h"
+
+// appleseed-max headers.
+#include "appleseedinteractive/appleseedinteractive.h"
+
+// Standard headers.
+#include <memory>
+
+namespace renderer { class Camera; }
+namespace renderer { class Project; }
+
+class ScheduledAction
+{
+  public:
+    virtual ~ScheduledAction() {}
+    virtual void update() = 0;
+};
+
+class CameraObjectUpdateAction : public ScheduledAction
+{
+  public:
+    explicit CameraObjectUpdateAction(
```

Comment:
You can remove `explicit` here since the argument takes two arguments. Again, it's not a mistake, just a minor styling thing.